code/modules/reagents/newchem/pyro.dm

```diff
@@ -417,6 +417,10 @@
 	for(var/turf/simulated/turf in range(min(created_volume/10,4),T))
 		new /obj/effect/hotspot(turf)
 
+/datum/reagent/phlogiston/reaction_mob(mob/living/M, method=TOUCH, volume)
+	M.IgniteMob()
+	..()
+
 /datum/reagent/phlogiston/on_mob_life(mob/living/M)
 	M.adjust_fire_stacks(1)
 	var/burndmg = max(0.3*M.fire_stacks, 0.3)
```

Comment:
To clarify, while the way you've changed it is a big improvement (it definitely needs to keep the ignite), it still functions very similarly to other chemicals in just applying damage per cycle, those are a dime a dozen.

Making it harder to roll out the fire depending on how much you used would keep the chemical focused on its unique effect so chemists are encouraged to not just use the bare minimum, depending on what they want to accomplish.